docs/src/test/java/docs/home/persistence/CqrsIntegrationTest.java

```diff
@@ -52,19 +51,25 @@
   static Injector injector;
   static Application application;
   static CassandraSession cassandraSession;
+  private FiniteDuration defaultTimeout = new FiniteDuration(18, TimeUnit.SECONDS);
 
   @BeforeClass
   public static void setup() throws Exception {
 
-    Config config = TestUtil.persistenceConfig("CqrsIntegrationTest", CassandraLauncher.randomPort());
+    Config config = TestUtil
+        .persistenceConfig("CqrsIntegrationTest", CassandraLauncher.randomPort())
+        // Following sets the default timeout on Akka TestProbes.
+        .withValue(
+            "akka.test.single-expect-default",
+            ConfigValueFactory.fromAnyRef("19s"));
```

Comment:
This is really kicking in!

I've tested locally with values like `300ms` or `1s` to force the error in non-travis environments.